src/SourceBuild/content/build.proj

```diff
@@ -70,7 +70,8 @@
                     HashCatalogFilePath="$(PoisonReportDataFile)"
                     MarkerFileName="$(PoisonMarkerFile)"
                     PoisonReportOutputFilePath="$(PoisonUsageReportFile)"
-                    NonShippingPackagesListFiles="@(NonShippingPackagesList)" />
+                    NonShippingPackagesListFiles="@(NonShippingPackagesList)"
+                    FailOnPoisonFound="true" />
```

Comment:
With the introduction of this, it seems like we can remove the PoisonTests and the associated baseline file.

Reply:
Great idea - will update the PR after local verification.

Reply:
Fixed with https://github.com/dotnet/installer/pull/16631/commits/73495fee352b64348f48f54ce2352acf341da357